src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -5383,22 +5389,17 @@ Gui::Action * CmdSketcherCompCreateFillets::createAction(void)
     return pcAction;
 }
 
-void CmdSketcherCompCreateFillets::updateAction(int mode)
+void CmdSketcherCompCreateFillets::updateAction(int __attribute__((unused))mode)

```

Comment:
It is the first time I see this one.

It does not seem to be in the standard:
https://en.cppreference.com/w/cpp/language/attributes

I read that apparently it works with gcc.

I will ask Werner:
@wwmayer 
Sorry. Me again. __attribute__((unused)) looks like not in the standard. My advice would be not to use it. What do you think?

Reply:
> Sorry. Me again. attribute((unused)) looks like not in the standard. My advice would be not to use it. What do you think?

Does it raise a warning? In FreeCAD we set the pedantic compiler flag to make sure to only allow strong ISO-compliant code and if a warning is raised it must be considered as bug and thus must be fixed.

From this description it's unclear whether it's a standard feature or a compiler extension: https://clang.llvm.org/docs/AttributeReference.html#maybe-unused-unused

Anyway, in all our current code where we have defined an unused variable we mark it with the Qt macro Q_UNUSED(mode).

Reply:
@emmanuelobrien

GCC does not complain. But I am not sure it is standard, so better stick to Q_UNUSED(mode). As Werner says, it is what we have always used and is proven to be portable.